astropy/coordinates/tests/test_angles.py

```diff
@@ -698,7 +698,7 @@ def test_latitude():
 
     lim_exc = r"Latitude angle\(s\) must be within -90 deg <= angle <= 90 deg, got"
     with pytest.raises(ValueError, match=rf"{lim_exc} \[91. 89.\] deg"):
-        Latitude(["91d", "89d"])
+        Latitude([91, 89] * u.deg)
```

Comment:
How does changing this contribute to fixing the bug?

Reply:
The error changes: because of the change in `Angle -> cls` above, if I keep a list of `str`, then the individual parsing of `Latitude("91d")` already causes an error. Since the test here was specifically meant to check a small array, I changed it. Obviously, this is another small API change...